restaurant/src/main/resources/properties/cloud.yml

```
@@ -38,4 +38,8 @@ eureka:
     register-with-eureka: true
     fetch-registry: true
     service-url:
-      defaultZone: ${PROD_EUREKA_SERVICE_URL}
\ No newline at end of file
+      defaultZone: ${PROD_EUREKA_SERVICE_URL}
+  instance:
+    hostname: ${PROD_EUREKA_INSTANCE_HOSTNAME}
+    prefer-ip-address: false
+    non-secure-port: 80
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)